server.js

```diff
@@ -109,53 +85,36 @@ async function postAccount(server) {
 }
 
 async function searchByCountry(server) {
-  const { CountryName, IndicatorName, Year } = await server.body;
-  // const { CountryName, Year } = await server.body;
-  console.log(CountryName);
-  if (CountryName && IndicatorName && Year) {
-    const result = (
-      await client.queryObject(
-        `SELECT Value FROM Indicators WHERE CountryName = $1 AND IndicatorName = $2 AND Year = $3`,
-        [CountryName, IndicatorName, Year]
-      )
-    ).rows;
-
-    console.log(result);
-    return server.json(result, 200);
-  } else if (CountryName && IndicatorName && !Year) {
-    console.log(IndicatorName);
-    const result = (
-      await client.queryObject(
-        `SELECT Year, Value FROM Indicators WHERE CountryName = $1 AND IndicatorName = $2 `,
-        [CountryName, IndicatorName]
-      )
-    ).rows;
+  const { country, indicator, year } = await server.body;
```

Comment:
It's a GET request so it won't have a body right? We'd need to get it from the header